resources/test/fixtures/flake8_simplify/SIM201_2.py

```diff
@@ -0,0 +1,2 @@
+if not a == b:    #NG when exception check
```

Comment:
Why is this special-cased? I'm assuming flake8-simplify special-cases it, just wondering what the reasoning is.

Reply:
It's in `flake8_simplify` 
code : [SIM201](https://github.com/MartinThoma/flake8-simplify/blob/d3cb9556a5255969f83a8329df8f635da41f595d/flake8_simplify/rules/ast_unary_op.py#L24-L25)
test: [not in exception check](https://github.com/MartinThoma/flake8-simplify/blob/d3cb9556a5255969f83a8329df8f635da41f595d/tests/test_200_rules.py#L10-L15)

I was wondering too, but implement it faithful the original code.

Reply:
Here's the PR that introduced it: https://github.com/MartinThoma/flake8-simplify/issues/13

No context given, but probably correct to preserve it for compatibility as you did here.

Reply:
My guess would be that the reason to keep the `not` is to maintain the parallel `if not cond: raise ...` has to `assert cond`.